lib/jbuild

```diff
@@ -8,7 +8,7 @@
 (rule
  ((targets (github_j.ml github_j.mli))
   (deps    (github.atd))
-  (action  (run atdgen -j ${<}))))
+  (action  (run atdgen -j-std ${<}))))
```

Comment:
You are correct! This seems like a bug in `atdgen`:

```
  -j-std
          Convert tuples and variants into standard JSON and
          refuse to print NaN and infinities (implying -json mode
          unless another mode is specified).
```

but with `-j-std` alone the files generated are called `foo.ml[i]` for `foo.atd` rather than `foo_j.ml[i]`.

I'm also a bit confused about why only a single CI instance caught this issue and the others seem to have completed successfully. 😕